install_config/redeploying_certificates.adoc

```diff
@@ -623,17 +623,67 @@ EOF
 oc create -f -
 ----
 +
-Then, run the following to remove the environment variables:
+Then, run the following command to remove the environment variables:
 +
 ----
 $ oc env dc/router OPENSHIFT_CA_DATA- OPENSHIFT_CERT_DATA- OPENSHIFT_KEY_DATA- OPENSHIFT_MASTER-
 ----
 
-. Delete the `router-certs` secret:
+. Obtain a certificate.
+* If you use an external Certificate Authority (CA) to sign your certificates,
+create a new certificate and provide it to {product-title} by following your
+internal processes.
+* If you use the internal {product-title} CA to sign certificates, run the
+following commands:
++
+[IMPORTANT]
+====
+The following commands generate a certificate that is internally signed. It will
+be trusted by only clients that trust the {product-title} CA.
+====
++
+----
+$ cd /root
+$ mkdir cert ; cd cert
+$ oc adm ca create-server-cert \
+    --signer-cert=/etc/origin/master/ca.crt \
+    --signer-key=/etc/origin/master/ca.key \
+    --signer-serial=/etc/origin/master/ca.serial.txt \
+    --hostnames='*.hostnames.for.the.certificate' \
+    --cert=router.crt \
+    --key=router.key \
+
+----
++
+These commands generates the following files:
++
+** A new certificate named *_router.crt_*.
+** A copy of the signing CA certificate chain, *_/etc/origin/master/ca.crt_*.
+This chain can contain more than one certificate if you use intermediate CAs.
+** A corresponding private key named *_router.key_*.
+
+. Create a new file that concatenates the generated certificates:
++
+----
+$ cat router.crt /etc/origin/master/ca.crt router.key > router.pem
+----
+
+. Back up the old certificate:
++
+----
+$ oc export secret router-certs > ~/old-router-certs-secret.yaml
+----
+
+. Create a new secret to hold the new certificate and key, and replace the 
+contents of the existing secret:
 +
 ----
-$ oc delete secret router-certs
+$ oc create secret tls router-certs --cert=router.pem \ <1>
+    --key=router.key -o json | \
+    oc replace -f -
```

Comment:
Given `oc delete ...` is removed, the command will fail, my result:
```
oc create secret tls ... -o json | oc replace -f -
Error from server (AlreadyExists): secrets ... already exists
```
You should keep `oc delete ...` and remove `-o json | oc replace -f -`:
```
$ oc delete secret router-certs
$ oc create secret tls router-certs ... --key=router.key
```
Or given `oc delete ...` removed, add `--dry-run`:
```
-$ oc delete secret router-certs
+$ oc create secret tls router-certs --cert=router.pem --key=router.key -o json --dry-run | oc replace -f -
```